### DISCUSSION TITLE
Trouble accessing state for all steps

### DISCUSSION THREAD

**Discussion by agentphoenix** [spatie/laravel-livewire-wizard#10] [Q&A] [answered]
Loving this package so far, so thank you for putting this out in the world!

One issue I've been having is accessing state from other steps. When I use `nextStep` to move forward in the wizard and then dump `$this->allStepsState()` it returns an array with a single item: the current step. The docs (and the code) seem to indicate this should have all the data from the other steps, but it doesn't appear to be working. Am I missing something obvious?

**Comment by freekmurze** [comment, marked as answer]
Thanks for reporting. This has been fixed in the latest release 👍
